pyproject.toml

```diff
@@ -9,7 +9,8 @@ authors = [
 dependencies = [
     "python-telegram-bot==20.8",
     "python-dotenv==1.0.1",
-    "SQLAlchemy==2.0.31"
+    "SQLAlchemy==2.0.31",
+    "pytest-xdist>=3.6.1",
```

Comment:
Typically you place pytest this in `dev_dependencies`

Reply:
made the change